dev/tasks/build/start.frag

```diff
@@ -0,0 +1,13 @@
+( function ( root, factory ) {
+	// Register the CKEDITOR global.
+	root.CKEDITOR = factory();
```

Comment:
I understand, but you must understand that this level of flexibility is required because we deal with all kinds of developers out there. Some would be prepared for AMD but many will just have troubles.

Note that our goal is not making CKEditor an AMD module, but instead **enable** it as a AMD module to satisfy AMD devs.

For the sake of being "right" we may even consider a configuration option for the builder where one could force "pure AMD", so no global is created. But by default we should go the "satisfy all" way for easy adoption.